slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -52,6 +52,40 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPositionNew(
+    ServerContext *context, const GetPositionNewRequest *request,
```

Comment:
I know this is all the same as the original, but in the interest of me learning:

why is `GetPositionNewRequest *request` `const`?

Reply:
never mind, I figured it out